### ISSUE TITLE
Listbox-Button: Support aria-invalid

### ISSUE THREAD

**Issue by ZZS77** [eBay/skin#741]
# Feature Request
Support aria-invalid for listbox

## Description
We have usecases where we show an error for these component types and want to apply aria-invalid="true" (with a red border on the element).

## Screenshots
Example in the current flow which we will apply to our new flow which will be DS6:
![image](https://user-images.githubusercontent.com/42783072/62322035-fad69780-b458-11e9-84ac-dbe62eef0215.png)

**Comment by seangates** [comment]
This will cause a breaking change in eBayUI Core, so we will have to wait until 9.0.0 (as 8.0.0) is almost out. /cc @ianmcburnie

**Comment by ianmcburnie** [comment]
This should be tackled as part of the relevant DS6.5 sprint.

**Comment by seangates** [comment]
Yup.

**Comment by ianmcburnie** [comment]
Ok, but you are saying this is going into 9.0.0?

**Comment by seangates** [comment]
Sorry, let me rephrase: 9.0.0 _or later_, whenever listbox is reviewd.

**Comment by ianmcburnie** [comment]
Done in 9.4.0
